.github/workflows/IntegrationTest.yml

```
@@ -14,8 +14,8 @@ jobs:
       fail-fast: false
       matrix:
         package:
-          - {user: TuringLang, repo: AdvancedPS.jl}
-          - {user: TuringLang, repo: Turing.jl}
+          - {user: TuringLang, repo: AdvancedPS.jl, ref: taped-libtask}
```

`AdvancedPS`‘s new releases is compatible with new `Libtask`

```suggestion
          - {user: TuringLang, repo: AdvancedPS.jl}
```

---

Not according to the logs of the AdvancedPS integration test run:
```
   Resolving package versions...
┌ Info: Not compatible with this release. No problem.
└   exception = empty intersection between Libtask@0.6.4 and project compatibility 0.5.3-0.5
```

---

It’s a bit strange - `AdvancedPS` @ v.3.2 support new versions of Libtask. In order to support the most recent Libtask release, I just updated compability bounds in AdvancedPS and made a new release. See https://github.com/JuliaRegistries/General/pull/53029

---

I looked into the CI message - it seems Integration test is using the `master` branch instead of the most recent release of `AdvancedPS` (i.e. v0.3.2). We currently have some breaking changes in `master` so the most recent release of AdvancedPS is based on the `releases` branch.